alea/blueice_extended_model.py

```diff
@@ -76,7 +78,13 @@ def get_expectation_values(self, **kwargs) -> dict:
         """
         ret = dict()
         for ll in self._likelihood.likelihood_list[:-1]:  # ancillary likelihood does not contribute
-            mus = ll(full_output=True, **kwargs)[1]
+
```

Comment:
because you're calling individual likelihood terms-- if one likelihood term but not another has a parameter you get an error (the LogLikelihoodSum does the filtering of arguments also)